src/towncrier/_project.py

```diff
@@ -38,47 +38,45 @@ def _get_package(package_dir: str, package: str) -> ModuleType:
 
 
 def get_version(package_dir: str, package: str) -> str:
-    module = _get_package(package_dir, package)
+    version: Any
```

Comment:
Why Any? Any opts `version` completely out of type checking, is there a rescue for that?

Reply:
changed to str #627